pkg/storage/export/export/vm-source.go

```diff
@@ -157,7 +157,7 @@ func (ctrl *VMExportController) isSourceInUseVM(vmExport *exportv1.VirtualMachin
 		}
 		return false, "", nil
 	}
-	return exists, "", nil
+	return exists, fmt.Sprintf("Virtual Machine Instance %s/%s does not exist", vmExport.Namespace, vmExport.Spec.Source.Name), nil
```

Comment:
I'm not sure this is the best place to return this message. The VMI not existing during a VMExport is actually the expected case as we don’t want to create the export server while the VM is running. So the ideal flow is: VM is stopped, VMI doesn’t exist, and we go ahead and create the export server. What we actually want to check here is whether the VM itself exists, probably somewhere in a place like https://github.com/kubevirt/kubevirt/blob/main/pkg/storage/export/export/vm-source.go#L189.

Reply:
Well, to be fair we support vmexports of standalone VMIs so we shoulnd't simply return the error in the line I shared above either. We would need to check if both the VM and the VMI doesn't exist and then return this message.